packages/web-components/src/components/va-radio/va-radio.tsx

```diff
@@ -250,7 +260,7 @@ export class VaRadio {
           aria-label={ariaLabel}
         >
           <legend part="legend">
-            {label}
+            {Header ? <Header part="header">{label}</Header> : label}
```

Comment:
I was in the React mind-set which requires user-defined components to be capitalized. So, probably better to rename this to `HeaderLevel` like you suggested

Reply:
That makes sense and yes, if you don't mind making this a little clearer (un-reactifying™️ it), that would be a quick improvement to readability. I would also prefer it to be camel case if there's no other reason to make it capitalized.

Reply:
I changed the variable to `headerLevel` and I guess it does behave like React. It ignored the value of the variable and rendered `<headerlevel>`

<img width="433" alt="rendered HTML in storybook showing 'headerlevel' (all lower-case) as the html tag name, not the expected h3" src="https://user-images.githubusercontent.com/136959/217587321-3109f9ca-5e4b-4135-a322-b986e763f1c8.png">

So, I'm going to switch to using `HeaderLevel` as that does render an `H3`

<img width="398" alt="rendered HTML in storybook showing an 'h3' rendered when using a captialized variable name" src="https://user-images.githubusercontent.com/136959/217588216-8180bddc-2a81-4f04-9e6c-73e851dd4d8c.png">

Reply:
Thanks for giving that a try. That's an interesting discovery!

Reply:
We could also rename this as `heading` instead of `header` as the[ h1-h6 are abbreviations to section `heading` levels](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) (to differentiate from the `header` landmark).